catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/ops/IcebergTableOpsHelper.java

```diff
@@ -143,7 +142,8 @@ private void doMoveColumn(
     } else if (columnPosition instanceof TableChange.First) {
       icebergUpdateSchema.moveFirst(DOT.join(fieldName));
     } else {
-      throw new NotSupportedException(
+      Preconditions.checkArgument(
+          columnPosition instanceof TableChange.Default,
```

Comment:
Does this change mean that if the user does not specify the column position, nothing will happen?

Reply:
no, Iceberg will treat it as `last` position.

Reply:
Iceberg will add the column to the `last` position, so no need to move the column

Reply:
I noticed that this method is called in two places, one is `doAddColumn` and the other is `doUpdateColumnPosition`. If it's called by `doUpdateColumnPosition`, no changes will be made either. Are you sure this meets expectations?

Reply:
yes, this seems bugy, let me think about it

Reply:
@mchades , fix it and add test, please help to review again